iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/MemoryLimitCheckTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.kubernetes.checks;
+
+import org.junit.jupiter.api.Test;
+import org.sonar.iac.common.api.checks.IacCheck;
+
+class MemoryLimitCheckTest {
+  IacCheck check = new MemoryLimitCheck();
+
+  @Test
+  void test_pod_object() {
```

Comment:
Test methods should be in camelCase.

Reply:
Let me know if this refactoring can be done for other checks as well in the current ticket (as it is only renaming of methods). I also changed the file names (eg: `test_pod_object.yaml` -> `pod_object.yaml`) after this, please let me know if I should revert this one

Reply:
Doesn't need to be, renaming test methods is enough.